Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2.7
+RUN apt update
+RUN apt upgrade -y
+RUN apt install vim -y
+RUN mkdir /sabot
+WORKDIR /sabot
+ADD . /sabot
```

Comment:
If you resort this, this could save some layers: 
`
ADD . /sabot
WORKDIR /sabot
`